keps/sig-node/2535-ensure-secret-pulled-images/README.md

```diff
@@ -332,7 +674,11 @@ TBD subsequent to alpha
   - Feature gate name: KubeletEnsureSecretPulledImages
   - Components depending on the feature gate: Kubelet
 - [x] Other
-  - Describe the mechanism: Kubelet configuration field `pullImageSecretRecheck`
+  - Describe the mechanism: Disabling is possible by disabling the kubelet feature gate
+    `KubeletEnsureSecretPulledImages`. This will turn off the feature completely.
+    An alternative is to set the `imagePullCredentialsVerificationPolicy` kubelet
+    configuration field to value `NeverVerify`, which will at least keep the pull
+    record caching, which would make it easier to turn on the feature later.
   - Will enabling / disabling the feature require downtime of the control
     plane?
     - No, only a restart of the kubelet
```

Comment:
Running pods will not be impacted though right?  A drain is not needed IIUC?

Reply:
correct, this only kicks in during the ensureimage step